scripts/utils/migrate/fetch-redirects.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const { paginate } = require('./docs-api');
+
+require('dotenv').config();
+
+const fetchAllRedirects = async () => {
+  const redirects = paginate('/api/migration/redirects', {
+    perPage: 300,
+  });
+
+  return groupRedirectsByDocsId(await redirects);
```

Comment:
```suggestion
  const redirects = await paginate('/api/migration/redirects', {
    perPage: 300,
  });

  return groupRedirectsByDocsId(redirects);
```

minor [nit] to move the `await` to the first expression instead of the 2nd.